app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -258,14 +257,25 @@ class TeamsAndRepositoriesConnector @Inject()(
       }
   }
 
-  def allRepositories(name: Option[String] = None, team: Option[String] = None, archived: Option[Boolean] = Some(false),
-                      repoType: Option[String] = None, serviceType: Option[String] = None)(implicit hc: HeaderCarrier): Future[Seq[GitRepository]] = {
-    val url = url"$teamsAndServicesBaseUrl/api/v2/repositories?name=$name&team=$team&archived=$archived&repoType=$repoType&serviceType=$serviceType"
+  def allRepositories(
+   name: Option[String] = None,
+   team: Option[TeamName] = None,
+   archived: Option[Boolean] = Some(false),
+   repoType: Option[RepoType] = None,
+   serviceType: Option[ServiceType] = None)
+             (implicit hc: HeaderCarrier): Future[Seq[GitRepository]] = {
+
+    val url = url"$teamsAndServicesBaseUrl/api/v2/repositories?name=$name&team=${team.map(_.asString)}&archived=$archived&repoType=${repoType.map(_.asString)}&serviceType=${serviceType.map(_.asString)}"
     httpClientV2
       .get(url)
       .execute[Seq[GitRepository]]
   }
 
+  def allBackendServices(implicit hc: HeaderCarrier): Future[Seq[GitRepository]] =
+    httpClientV2
```

Comment:
can this just call `allRepositories(repoType = Some(RepoType.Service), serviceType = Some(ServiceType.Backend)` - or the client could just do this and remove this function?

Reply:
this is some left over code I missed, going to remove it